huntflow_api_client/models/common.py

```diff
@@ -5,5 +5,5 @@
 
 
 class JsonRequestModel(BaseModel):
-    def jsonable_dict(self) -> Dict[str, Any]:
-        return json.loads(self.json())
+    def jsonable_dict(self, exclude_none: bool = False) -> Dict[str, Any]:
```

Comment:
It seems you need to copy all the parameters of the `BaseModel.json()` method and pass them to `json` method
```python
class JsonRequestModel(BaseModel):
    def jsonable_dict(
        self,
        *,
        include: Union['AbstractSetIntStr', 'MappingIntStrAny'] = None,
        exclude: Union['AbstractSetIntStr', 'MappingIntStrAny'] = None,
        by_alias: bool = False,
        skip_defaults: bool = None,
        exclude_unset: bool = False,
        exclude_defaults: bool = False,
        exclude_none: bool = False,
        encoder: Optional[Callable[[Any], Any]] = None,
        **dumps_kwargs: Any,
    ) -> Dict[str, Any]:
        params = {
            "include": include,
            "exclude": exclude,
            "by_alias": by_alias,
            "skip_defaults": skip_defaults,
            "exclude_unset": exclude_unset,
            "exclude_defaults": exclude_defaults,
            "exclude_none": exclude_none,
            "encoder": encoder,
        }
        return json.loads(self.json(**params, **dumps_kwargs))
```